packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -914,6 +914,7 @@ void main() {
 
     tester.testTextInput.log.clear();
     tester.testTextInput.closeConnection();
+    await tester.pump();
```

Comment:
why additional pump?

Reply:
A pump is needed to allow the focus change (unfocus) to be resolved.